app/views/candidate_mailer/application_submitted.text.erb

```diff
@@ -1,13 +1,23 @@
 Dear <%= @application_form.first_name %>,
 
-You have submitted an application for:
+You have submitted an application for <%= @application_choice.course_option.course.name_and_code %> at <%= @application_choice.course_option.course.provider.name %>.
+
+Sign in to your account to check the progress of your application:
+
+<%= @candidate_magic_link %>
+
+<% if @application_form.application_choices.count > 1 %>
+
+  If your training providers decide to progress your application, they’ll contact you to organise an interview.
```

Comment:
if the application choices count is one, shouldn't this first line be the singular, and the line after else the plural? @avinhurry

Reply:
It's if the count is greater than `1 (>)`, so if it is greater than 1 it would output the line of text below it. 

So that means we default to the singular for anything else, ie (potential bugs where the count ends up being `nil`) which kind of made more sense to me, but happy to change it around if you think we should. Your suggestion is probably more readable.

Reply:
ahhhh didn't see the greater than sign - thanks Avin!

Reply:
If they've submitted the application with more than one choice, then the first line ("You have submitted an application for X at Y") doesn't make sense?

Reply:
when it's more than one I think the line reads: 

You have submitted an application for: [list course names and provider names]

(so it's 1 application, for multiple courses/providers)

We're not consistent with application/applications in the service - something to look at in the future

Reply:
@EmmaFrith I’m not sure the code in this PR right now does that - the bullet list has been removed and line 3 just refers to a single course? Unless I missing something... @avinhurry ?

Reply:
Yeah good spot @frankieroberto, you are right. `You have submitted an application for...` (singular) isn't accurate if they have submitted more than one application. 

Do we need to make that line conditional too? Can we update the doc with what we want it to be please. https://docs.google.com/document/d/1v8eGxJuWU0JGoP71zT2sgFuBRpqO80V4-oHpTaYi6f8/edit

Reply:
You have submitted an application for the following courses:

[list courses and providers applied for]

could work 


---

We could change it to 'applications' if it's plural but if we do that we should do that throughout the entire email. We use 'application' quite a lot. E.g. one application for multiple courses.

Reply:
@avinhurry I don't just mean the pluralisation - doesn't line 3 currently only list a single course?

```ruby
You have submitted an application for <%= @application_choice.course_option.course.name_and_code %> at <%= @application_choice.course_option.course.provider.name %>.
```

?

Reply:
Discussed offline - the decision is to leave line 3 in for singular and render the loop which was previously here if its plural.

Reply:
Changes made @EmmaFrith

Reply:
thanks @avinhurry!